_studio/mfx_lib/mctf_package/mctf/src/mctf_common.cpp

```diff
@@ -365,6 +365,11 @@ mfxStatus CMC::MCTF_RELEASE_FRAME(
     {
         MFX_CHECK(mco, MFX_ERR_UNDEFINED_BEHAVIOR);
         mco = nullptr;
+
+        device->DestroySurface(QfIn[0].frameData);
```

Comment:
If that's where we destroy surface created in avc encoder, then I don't like the design. MCTF kind of accepted input surface and destroyed it internally, That's not good... I think component which allocated memory should (optimally) be responsible for it destruction.

Reply:
Fixed